docs/examples.rst

```diff
@@ -101,6 +101,13 @@ To pass additional options to Kerberos auth use dict ``kerberos_options``, e.g.:
 
     auth_jira = JIRA(kerberos=True, kerberos_options={'mutual_authentication': 'DISABLED'})
 
+Authenticating on a remote server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You may also pass multiple arguments to initialize the JIRA object. For example, to log into a remote Jira server with basic authentication::
+
+   jira = JIRA('https://jira.atlassian.com', basic_auth('username', 'password'))
```

Comment:
Basic auth example is a few sections above, by the way on line 52. 

You can modify to also pass in the server, but otherwise, there is no need for duplicate basic auth examples.